src/legacy/core_plugins/data/public/expressions/expressions_service.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Ast } from '@kbn/interpreter/common';
+
+// TODO:
+// this type import and the types below them should be switched to the types of
+// the interpreter plugin itself once they are ready
+import { RunPipelineHandlers } from 'ui/visualize/loader/pipeline_helpers/run_pipeline';
```

Comment:
nit: can you for now just duplicate that code here, i will be removing run_pipeline helper as soon as this gets merged